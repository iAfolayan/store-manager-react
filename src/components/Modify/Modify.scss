img.rounded.mx-auto.d-block {
  width: 3rem;
}