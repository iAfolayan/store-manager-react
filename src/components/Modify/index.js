import React, { Component, Fragment} from 'react';
import { NavLink } from 'react-router-dom';
import prodImage5 from '../../images/5.PNG';
import './Modify.scss';

class Modify extends Component {

  render() {
    return (
     <Fragment>
        <h5 className="ml-3 text-primary">Products</h5>
        <hr className="mr-5" />
      <div className="table-responsive-sm">
        <table className="table table-bordered table-hover">
          <thead>
            <tr>
              <th scope="col">#</th>
              <th scope="col">Name</th>
              <th scope="col">category</th>
              <th scope="col">price</th>
              <th scope="col">Image</th>
              <th scope="col">ACTION</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <th scope="row">1</th>
              <td>Bluetooth</td>
              <td>Phone</td>
              <td>3000</td>
              <td>
                <img src={prodImage5} className="rounded mx-auto d-block" alt="image title" />
              </td>
              <td>
                <NavLink to="edit">Edit</NavLink> | Delete</td>
            </tr>
            <tr>
              <th scope="row">2</th>
              <td>Bluetooth</td>
              <td>Phone</td>
              <td>3000</td>
              <td>
                <img src={prodImage5} className="rounded mx-auto d-block" alt="image title" />
              </td>
              <td>
                <NavLink to="edit">Edit</NavLink> | Delete</td>
            </tr>
            <tr>
              <th scope="row">3</th>
              <td>Bluetooth</td>
              <td>Phone</td>
              <td>3000</td>
              <td>
                <img src={prodImage5} className="rounded mx-auto d-block" alt="image title" />
              </td>
              <td>
                <NavLink to="edit">Edit</NavLink> | Delete</td>
            </tr>
            
          </tbody>
        </table>
      </div>
    </Fragment>
    );
  }
}

export default Modify;