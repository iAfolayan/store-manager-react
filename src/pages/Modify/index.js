import React, { Fragment } from 'react';
import Footer from '../../components/Footer';
import Navigation from '../../components/Navbar';
import Modify from '../../components/Modify';

const ModifyHome = () => {
  return (
    <Fragment>
      <div className="form-group row">
        <div className="col-md-2">
          <Navigation />
        </div>
        <div className="col-md-10">
          <Modify />
          <Footer footerClassName="footer inner" />
        </div>
      </div>
    </Fragment>
  );
}

export default ModifyHome;
