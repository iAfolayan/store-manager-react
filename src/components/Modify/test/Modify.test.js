import React from 'react';
import { shallow } from 'enzyme';
import Modify from '..';

describe('<Modify product />', () => {
  test('should rendered Modify Component without crashed', () => {
    const wrapper = shallow(<Modify />);
    expect(wrapper).toMatchSnapshot();
  });
});