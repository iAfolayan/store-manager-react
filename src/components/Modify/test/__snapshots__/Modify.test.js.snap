// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Modify product /> should rendered Modify Component without crashed 1`] = `
<Fragment>
  <h5
    className="ml-3 text-primary"
  >
    Products
  </h5>
  <hr
    className="mr-5"
  />
  <div
    className="table-responsive-sm"
  >
    <table
      className="table table-bordered table-hover"
    >
      <thead>
        <tr>
          <th
            scope="col"
          >
            #
          </th>
          <th
            scope="col"
          >
            Name
          </th>
          <th
            scope="col"
          >
            category
          </th>
          <th
            scope="col"
          >
            price
          </th>
          <th
            scope="col"
          >
            Image
          </th>
          <th
            scope="col"
          >
            ACTION
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <th
            scope="row"
          >
            1
          </th>
          <td>
            Bluetooth
          </td>
          <td>
            Phone
          </td>
          <td>
            3000
          </td>
          <td>
            <img
              alt="image title"
              className="rounded mx-auto d-block"
              src="test-file-stub"
            />
          </td>
          <td>
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="edit"
            >
              Edit
            </NavLink>
             | Delete
          </td>
        </tr>
        <tr>
          <th
            scope="row"
          >
            2
          </th>
          <td>
            Bluetooth
          </td>
          <td>
            Phone
          </td>
          <td>
            3000
          </td>
          <td>
            <img
              alt="image title"
              className="rounded mx-auto d-block"
              src="test-file-stub"
            />
          </td>
          <td>
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="edit"
            >
              Edit
            </NavLink>
             | Delete
          </td>
        </tr>
        <tr>
          <th
            scope="row"
          >
            3
          </th>
          <td>
            Bluetooth
          </td>
          <td>
            Phone
          </td>
          <td>
            3000
          </td>
          <td>
            <img
              alt="image title"
              className="rounded mx-auto d-block"
              src="test-file-stub"
            />
          </td>
          <td>
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="edit"
            >
              Edit
            </NavLink>
             | Delete
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</Fragment>
`;
